README.md

```diff
@@ -44,10 +44,10 @@ const config: NodeConfig = {
     "enode://XXXX@X[::]:XXXX",
     "enode://YYYY@Y[::]:YYYY"
   ],
-  "networkID": networkID, // --networkid / Network identifier (integer, 0=Olympic (disused), 1=Frontier, 2=Morden (disused), 3=Ropsten) (default: 1)
+  "networkID": networkID, // --networkid / Network identifier (integer, 42220=mainnet, 62320=baklava (testnet), 44787=alfajores (testnet)) (default: 1)
   "maxPeers": 0, // --maxpeers / Maximum number of network peers (network disabled if set to 0) (default: 25)
   "genesis": genesis, // genesis.json file
-  "nodeDir": ".private-ethereum", // --datadir / Data directory for the databases and keystore
+  "nodeDir": ".celo", // --datadir / Data directory for the databases and keystore
```

Comment:
Thinking out loud, we'll want to make sure an update from Valora 1.0.1 to 1.1.0 with these changes doesn't throw errors due to the changed nodeDir. Agree we should make this change here though 👍

Reply:
Fwiw the approach the app does is specifying `.${DEFAULT_TESTNET}` as the nodeDir, I think (not totally sure) it tolerates a change well